Makefile

```diff
@@ -26,6 +26,8 @@ help:
 	@echo "Secondary targets:"
 	@echo
 	@echo "- lint			Run flake8 checker on the Python code"
+	@echo "- upgrade		Upgrade the dependencies."
+	@echo "- upgrade-dev	Upgrade the dev. dependencies."
```

Comment:
By the way, I think there are indentation problems in the UI makefile usage tips: https://github.com/c2corg/v6_ui/blob/master/Makefile#L26-L37
Do you how it should be formatted? Using a fixed number of tabs?